app/components/Popover/Popover.css

```diff
@@ -1,11 +1,7 @@
+@import '~app/styles/variables.css';
+@import '~app/styles/overlay.css';
+
 .content {
-  background: var(--color-white);
-  border-radius: 5px;
-  border: 1px solid var(--color-mono-gray-3);
-  box-shadow: 0 4px 2px var(--lego-color-gray-light);
-  position: absolute;
   max-width: 500px;
   padding: 20px;
-  z-index: 1000;
```

Comment:
Ooo that's a terrifying removal 😬 But I'm assuming it was greatly tested

Reply:
No idea why it was that high. Popover is only used in one place, on the event calendar and works fine with the same z-index as Dropdown :)